integration/integration_tests.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+# Basic binary compatibility test script for replace_landsurface.
+# See INTEGRATION_README.md for details on test usage, data and options.
+
+TEST_DATA_DIR=/g/data/vk83/testing/data/replace_landsurface/integration_tests
+INPUT_DIR=$TEST_DATA_DIR/input_data
+OUTPUT_DIR=$TEST_DATA_DIR/expected_outputs
+DRIVING_DATA_DIR=$TEST_DATA_DIR/driving_data
+CLEAN_OUTPUT=true
+
+declare -A pids # Associative array (similar to a dictionary in Python)
+
+# Create a temporary work directory
+WORK_DIR=$(mktemp -d)
+echo -e "Work directory: $WORK_DIR\n"
```

Comment:
See above. I'd advise using curly braces, but it's a style thing and I'm not going to insist, just suggest.